interface/main/onotes/office_comments.php

```diff
@@ -72,26 +71,22 @@
         } else {
             $date_string = $date;
         }
-
             $card  = '';
-            $card .= '<div class="panel panel-default">';
-            $card .= '    <div class="panel-heading">';
+            $card .= '<thead><th align="center">';
             $card .= '        <h3 class="panel-title">'.text($date_string).' <strong>('.text($note->getUser()->getUsername()).')</strong></h3>';
-            $card .= '    </div>';
-            $card .= '    <div class="panel-body">';
-            $card .=          nl2br(text($note->getBody()));
-            $card .= '    </div>';
-            $card .= '</div>';
-
+            $card .= '</th></thead>';
+            $card .= '<tr><td>';
+            $card .= nl2br(text($note->getBody()));
+            $card .= '</th></td>';
+            $card .= '';
```

Comment:
Since they were not inside a bootstrap tag, I didn't know the panel-default  panel-heading  panel-body are bootstrap themes. I'll put them back